vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -110,10 +151,11 @@ def pre_build(self):
         #  Or just don't do validation at pre_build time and wait until state is available during build time instead?
         #  What should the load kwargs be here? Remember they need to be {} for static data.
         #  Note that currently _get_unfiltered_data is only suitable for use at runtime since it requires
-        #  ctd_parameters. That could be changed to just reuse that function.
+        #  ctds_parameter. That could be changed to just reuse that function.
         multi_data_source_name_load_kwargs: list[tuple[DataSourceName, dict[str, Any]]] = [
             (model_manager[target]["data_frame"], {}) for target in proposed_targets
         ]
```

Comment:
It's possible to send default DFP values from the model_manager into the `data_manager._multi_load()` with the following code: 
(Disclaimer: The code is just a PoC and could be optimised a lot)

```py
        from vizro.models._controls import Parameter

        multi_data_source_name_load_kwargs: list[tuple[DataSourceName, dict[str, Any]]] = []

        # One tuple per filter.target
        # [
        #     ('data_1', {'arg_1': 1, 'arg_2': 2,}),
        #     ('data_2', {'X': "ASD"}),
        #     ('data_2', {'X': "qwe"}),
        # ]

        # TODO-NEXT: The code below is just a PoC and could be improved a lot.
        page_obj = model_manager[model_manager._get_model_page_id(model_id=ModelID(str(self.id)))]
        for target in proposed_targets:
            data_source_name = model_manager[target]["data_frame"]
            load_kwargs = {}

            for page_parameter in page_obj.controls:
                if isinstance(page_parameter, Parameter):
                    for parameter_targets in page_parameter.targets:
                        if parameter_targets.startswith(f'{target}.data_frame'):
                            argument = parameter_targets.split('.')[2]
                            # argument is explicitly defined
                            if parameter_value := getattr(page_parameter.selector, 'value', None):
                                load_kwargs[argument] = parameter_value
                            # find default value
                            else:
                                parameter_selector = page_parameter.selector
                                default_parameter_value = None
                                if isinstance(parameter_selector, Dropdown):
                                    default_parameter_value = parameter_selector.options if parameter_selector.multi else parameter_selector.options[0]
                                elif isinstance(parameter_selector, Checklist):
                                    default_parameter_value = parameter_selector.options
                                elif isinstance(parameter_selector, RadioItems):
                                    default_parameter_value = parameter_selector.options[0]
                                elif isinstance(parameter_selector, Slider):
                                    default_parameter_value = parameter_selector.min
                                elif isinstance(parameter_selector, RangeSlider):
                                    default_parameter_value = [parameter_selector.min, parameter_selector.max]

                                load_kwargs[argument] = default_parameter_value

            multi_data_source_name_load_kwargs.append((data_source_name, load_kwargs))
```

Reply:
The comment you added here is perfect, thanks @petar-qb.